proxies/dotnet/Controllers/Client.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Mvc;
+using DevCycle.SDK.Server.Cloud.Api;
+using DevCycle.SDK.Server.Local.Api;
+using DevCycle.SDK.Server.Common.Model.Local;
+using DevCycle.SDK.Server.Common.Model.Cloud;
+using System.Text.Json.Serialization;
+
+namespace dotnet.Controllers;
+
+public class ClientOptions: DVCLocalOptions
+    {
+        [JsonPropertyName("configPollingIntervalMs")]
+        new public int ConfigPollingIntervalMs { get; set; }
+
+        [JsonPropertyName("eventFlushIntervalMS")]
+        new public int EventFlushIntervalMs { get; set; }
+
+        [JsonPropertyName("baseURLOverride")]
+        public string? BaseURLOverride { get; set; }
+
+        [JsonPropertyName("enableEdgeDB")]
+        public bool? EnableEdgeDB { get; set; }
+    }
+
+public class ClientRequestBody {
```

Comment:
should we put these in a models folder

Reply:
personally I prefer it to be colocated with the place its actually going to be used since its not really a "model" thats used anywhere else, it's specifically the body of this one request

Reply:
it might be convenient for this file since it's a small request body but for e.g. my pr where the user request can be a big request body, it made more sense for me to separate it to have the file more readable, so would I then put it in the file even if it makes it larger / harder to read?

Reply:
I guess in that case its fine to have it separate, like our DTOs in Nest.

However in Nest we still locate the files defining those DTOs within the same "module" so that theyre easy to find relative to the other files that define that set of endpoints.